webapp/channels/src/components/channel_view/channel_view.tsx

```diff
@@ -159,6 +159,31 @@ export default class ChannelView extends React.PureComponent<Props, State> {
                     <AdvancedCreatePost/>
                 </div>
             );
+        } else {
+            createPost = (
+                <div
+                    className='post-create__container'
+                    id='post-create'
+                >
+                    <div
+                        className='channel-archived__message'
+                    >
+                        <FormattedMarkdownMessage
+                            id='noRolesMessage'
+                            defaultMessage='Roles for this channel are not yet loaded. Please wait or reload the app.'
```

Comment:
This would be up to Abhijit, but I don't think we need to tell the user exactly what's happening here since they wouldn't know what to do with that information anyway. If we had a skeleton component to use here, that'd be good, but I think it'd be fine to either show the standard loading indicator or even just render nothing since this should ideally be gone before the user ever sees it